src/@next/Tooltip/Tooltip.tsx

```diff
@@ -247,39 +247,45 @@ export const Tooltip = ({
       content
     );
 
-  const handleMouseEnter = () => setIsActive(true);
+  const handleMouseEnter = () => {
+    if (!clickable) setIsActive(true);
+  };
   const handleMouseLeave = () => {
-    setIsActive(false);
-    handleClick();
+    if (!clickable) {
+      setIsActive(false);
+      handleClick();
+    }
   };
 
   const timeoutRef = useRef<NodeJS.Timeout | null>(null);
   const [animate, setAnimate] = useState<boolean>(false);
   const handleClick = () => {
-    onClick();
-    // if you click during the tooltip's lifespan, it should reset the timeout
-    if (timeoutRef.current) {
-      clearTimeout(timeoutRef.current);
-      timeoutRef.current = null;
-      setAnimate(false);
-    }
-    setIsActive(true);
-    timeoutRef.current = setTimeout(() => {
-      setAnimate(true);
-      timeoutRef.current = setTimeout(() => {
-        setIsActive(false);
+    if (clickable) {
+      onAction?.();
+      // if you click during the tooltip's lifespan, it should reset the timeout
+      if (timeoutRef.current) {
+        clearTimeout(timeoutRef.current);
+        timeoutRef.current = null;
         setAnimate(false);
-      }, 370); // animation is 400ms, make this slightly lower to prevent visual glitch
-    }, timeout);
+      }
+      setIsActive(true);
+      timeoutRef.current = setTimeout(() => {
+        setAnimate(true);
+        timeoutRef.current = setTimeout(() => {
+          setIsActive(false);
+          setAnimate(false);
+        }, 370); // animation is 400ms, make this slightly lower to prevent visual glitch
+      }, timeout);
```

Comment:
sorry i didn't saw the specs for animations. If we'll have a static time, we won't need the timeout prop and these lines. you can apply the fade in and out in styles instead.

Reply:
the animation is static, the timeout is just how long the icon will be there after we click it before it proceeds showing the fade out animation and disappear

Reply:
or you can pass the time on how long should it be present in the animations instead or compute for it